spec/blueberry_rails_spec.rb

```diff
@@ -1,65 +1,27 @@
 require 'minitest/autorun'
-require 'fileutils'
+require './spec/support/blueberry_rails_helpers.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.